packages/dht/src/connection/websocket/WebsocketConnector.ts

```diff
@@ -159,12 +163,22 @@ export class WebsocketConnector {
                         this.attachHandshaker(connection)
                     } else {
                         logger.trace('incoming Websocket connection before localPeerDescriptor was set, closing connection')
-                        connection.close(false).catch(() => {})
+                        connection.close(false).catch(() => { })
                     }
                 }
             })
             const port = await this.websocketServer.start()
             this.selectedPort = port
+
+            if (this.config.geoIpDatabaseFolder) {
```

Comment:
Could encapsulate the geoip-location related logi to a facade module.

Reply:
Perhaps it would make sense to start the geoip-location before setting the server listener?

Reply:
Changed the starting order, I don't think there is a need for a facade for such a simple component as this.